### ISSUE TITLE
Add Scene Summary and Scene Full generation UI

### ISSUE THREAD

**Issue by headchem** [headchem/StoryGhostPlotter#103]
Set up Admin section to generate finetune files for both Scene Summary and Scene Full. Follow pattern of blurbs/expanded for how to include history and prompts.

Same as Blurbs and Expanded Summary, add UI for brainstorming Scene Summary and Scene Full (in the complex UI).

Then add the CYOA interface to the Simple UI.

Don't put new line breaks in the prompt for now, so the older models will still work. We can retrain all the models with line breaks once we've gathered more data.

**Comment by headchem** [comment]
Option parameters could be desired Characters and start/end emotion for a scene. Characters can be force in by increasing the logits on Character names (how to handle multiple complex tokens that make up a single name?) For emotion, we rely on the training data to have learned how a scene starts->ends with the labeled emotions.
